### PR TITLE
able to show multiple questions fixes #29

### LINKED ISSUE #29: Multiple questions ability
The system must be able to show multiple questions. Thus the HTML must have a variable part which has the question in it. Refer to Bottle's documentation on the template system. We will be using that.

During attempts also, there must be a field which mentions which question the person is attempting so that the given output can be checked against the correct "expected.txt" file.

### PR DESCRIPTION
**Pull request by Shivank98**
this fixes #29

### CODE REVIEW

**Comment by theSage21** [comment]
It's doing more than fixing #29

**Comment by Shivank98** [comment]
yes, it's working. I already checked.
but in order to check the output, I will need something like #39.

File: files/questions/1/statement.html [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+		</div>
+
+		<div class="row">
+			<div class="column column-20">
+				
+			</div>
+			<div class="column submit-column">
+				<div class="shift-center">
+					<!--Submission-->
+					<form action="/upload", method = "post", enctype = "multipart/form-data">
```
Comment (theSage21) [inline files/questions/1/statement.html]:
```suggestion
					<form action="/question/1/upload", method = "post", enctype = "multipart/form-data">   
```

File: index.html [hunk 2]
```diff
@@ -36,7 +36,7 @@ <h1>Submission Page</h1>
 			<div class="column submit-column">
```
Comment (theSage21) [inline index.html]: This file needs to be deleted.

Comment (Shivank98) [reply index.html]: can I change in next PR, I will be making some changes in it. like using it as a homepage.

Comment (theSage21) [reply index.html]: You can but it's a bad thing.

- It does not look like you forked this branch from master (probably from your existing branch)
- Those commits will end up being duplicated in the master when it is finally merged.

Best thing would be that you fork a new clean branch, code, and submit that in a new PR after closing this one.

Comment (Shivank98) [reply index.html]: ok closing this one. this time one issue at a time

**Pull request closed without merging**